### DISCUSSION TITLE
How do I download the jar files

### DISCUSSION THREAD

**Discussion by Scorpionadi** [nasko222/the-minecraft-recreation-project#132] [Q&A] [answered]
Hi nasko where can I download the jar files pls tell

**Comment by nasko222** [comment, marked as answer]
Click on instructions.md

**Comment by nasko222** [comment]
But overall the jar files are located in different branches of the repo

Think about it like windows folders, the jar files are in different folders

**Comment by ymokhless** [comment]
hehehahahahahahahahahah
